src/longopt/dimfilter_inc.h

```diff
@@ -24,39 +24,20 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'Q', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'S', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'T', "",
-	          "",                  "",
-	          "",                  "" },
+	{ 0, 'D', "distance",     "", "", "", "" },
+	{ 0, 'F', "filter",
+	          "b,c,g,m,p",    "boxcar,cosarch,gaussian,median,maxprob",
+	          "l,u",          "lower,upper" },
+	{ 0, 'G', "outgrid",
+	          "",             "",
+	          "d,n,o,s",      "divide,nan,offset,scale" },
+	GMT_INCREMENT_KW,         /* quasi-common -I option from gmt_constants.h */
+	{ 0, 'L', "script", "", "", "", "" },
+	{ 0, 'N', "secondary_filter|secfilter",
```

Comment:
PyGMT uses `sectors` for `-N`, but it must be a mistake.

Reply:
Not really. Turns out there is a sector setting as part of the filter setting and it does a filtering of the results across the sectors.  I renamed it sector_filter with alias secfilter